debounce-batch.js

```diff
@@ -19,21 +19,21 @@ module.exports = class DebouncingBatchAdd {
     this.queueByAuthor.delete(authorId)
     this.timestampsByAuthor.delete(authorId)
     // Add the messages in the queue
-    this.addBatch(msgVals, (err, kvts) => {
+    this.addBatch(msgVals, queue[0][1], (err, kvts) => {
```

Comment:
could we do something like const opts = queue[0][1]. And then pass that in here. Would make things easier to read.

Reply:
done